apps/ledger-live-mobile/src/components/Web3AppWebview/liveSDKLogic.ts

```diff
@@ -1,14 +1,19 @@
 import { isTokenAccount } from "@ledgerhq/live-common/account/index";
 import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
 import { Transaction } from "@ledgerhq/live-common/generated/types";
-import { Account, AccountLike, TransactionCommon } from "@ledgerhq/types-live";
+import { Account, AccountBridge, AccountLike, TransactionCommon } from "@ledgerhq/types-live";
+import { bridge as ACREBridge } from "@ledgerhq/live-common/families/bitcoin/ACRESetup";
 
 export default function prepareSignTransaction(
   account: AccountLike,
   parentAccount: Account | undefined,
   liveTx: Partial<Transaction>,
+  isACRE?: boolean,
 ): TransactionCommon {
-  const bridge = getAccountBridge(account, parentAccount);
+  const bridge = isACRE
+    ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (ACREBridge.accountBridge as unknown as AccountBridge<any>)
+    : getAccountBridge(account, parentAccount);
```

Comment:
I don't think we actually need this change as it's not doing anything special with the `hw-app-acre`